### PR TITLE
Storage release 11-27-2023

### PR DESCRIPTION
**Pull request by danieltprice**
Release notes preview:
https://neon-next-git-dprice-storage-11-27-2023-neondatabase.vercel.app/docs/release-notes/2023-11-27-storage-and-compute

Accompanying doc update for neon_superuser:
https://neon-next-git-dprice-storage-11-27-2023-neondatabase.vercel.app/docs/manage/roles#the-neonsuperuser-role

neon extension doc (we should mention it in the docs if we're mentioning it in the release notes)
https://neon-next-git-dprice-storage-11-27-2023-neondatabase.vercel.app/docs/extensions/neon

### CODE REVIEW

File: content/release-notes/2023-11-27-storage-and-compute.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+description: Read about the latest fixes and improvements in Neon Serverless Postgres.
```
Comment (bgrenon) [inline content/release-notes/2023-11-27-storage-and-compute.md:2]: Do we want to use this description to highlight a given feature/fix per release? In aggregate, in an RSS reader, more targetted descriptions might help with scanning the list. Example:
image

File: content/docs/manage/roles.md [hunk 2]
```diff
@@ -39,6 +39,8 @@ Roles created in the Neon console, CLI, or API, including the default role creat
 
 You can think of roles with `neon_superuser` privileges as administrator roles. If you require roles with limited privileges, such as a read-only role, you can create those roles from an SQL client. For more information, see [Manage database access](/docs/manage/database-access).
 
+Creating a database with the `neon_superuser` role using `CREATE DATABASE dbname WITH OWNER neon_superuser` syntax is not permitted. This `NOLOGIN` role should not be used directly or modified.
```
Comment (bgrenon) [inline content/docs/manage/roles.md:42]:
```suggestion
<Admonition type="note">
Creating a database with the `neon_superuser` role using `CREATE DATABASE dbname WITH OWNER neon_superuser` syntax is _not_ permitted. This `NOLOGIN` role should not be used directly or modified.
</Admonition>
```

**Review by bgrenon — APPROVED** [review]
Minor comments and possible discussion point re: descriptions.

File: content/release-notes/2023-11-27-storage-and-compute.md (hunk 1 above)
Comment (danieltprice) [reply content/release-notes/2023-11-27-storage-and-compute.md]: Good point. Thank you. I have revised the description.

**Review by lubennikovaav — APPROVED** [review]

**Pull request merged**